server/lib/app.coffee

```diff
@@ -28,8 +30,14 @@ class exports.App
         if @app.scripts?.start?
             @app.server = @app.scripts.start
         else
-            manifest = require path.join(@app.dir, "package.json")
-            if manifest.scripts?.start?
+            try
+                manifest = require path.join(@app.dir, "package.json")
+            catch
+                if @app?.name?
```

Comment:
@poupotte why `@app?` ? We are sure that `@app` exists here. Or Am I missing something?